code/__defines/species.dm

```diff
@@ -0,0 +1,95 @@
+#define SPECIES_HUMAN				"Human"
```

Comment:
Very slight nitpick that doesn't have to be addressed, but evening out the spacing here would make my eyes feel nice.

Reply:
The spacing is consistent when viewed in dreammaker.